SetupDataPkg/ConfDataSettingProvider/ConfDataSettingProvider.c

```diff
@@ -802,6 +975,13 @@ SettingsProviderSupportProtocolNotify (
     DEBUG ((DEBUG_ERROR, "Failed to Register.  Status = %r\n", Status));
   }
 
+  // register runtime settings provider
+  DEBUG ((DEBUG_INFO, "Registering runtime Setting Provider\n"));
+  Status = mSettingProviderProtocol->RegisterProvider (mSettingProviderProtocol, &mRuntimeSettingsProvider);
+  if (EFI_ERROR (Status)) {
+    DEBUG ((DEBUG_ERROR, "Failed to Register Runtime Settings.  Status = %r\n", Status));

```

Comment:
Are we going to continue this flow if this failed?

Reply:
I followed what the configuration settings provider does, which is continue. I think we don't need to fail boot in this case, it will just prevent the full runtime settings from getting applied during runtime, but wouldn't affect normal boot up.